packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -471,17 +475,20 @@ class _SummaryCardState extends State<SummaryCard> {
     final List<Widget> attributeChips,
   ) {
     _totalPrintableRows -= (attributeChips.length / 2).ceil();
-    return Column(
-      children: <Widget>[
-        header,
-        Container(
-          alignment: Alignment.topLeft,
-          child: Wrap(
-            runSpacing: 16,
-            children: attributeChips,
+    return AbsorbPointer(
```

Comment:
Nice widget havn't seen it before, wouldn't it make sense to put it around the whole summary card (below the widget which opens the full product page)

Reply:
Oh okay makes sense, feel free to merge